entries/h/hellobonsai.com.json

```
@@ -0,0 +1,12 @@
+{
+    "Bonsai": {
+      "domain": "hellobonsai.com",
+      "tfa": [
+        "email",
+        "sms"
+      ],
```

It's better than nothing... :man_shrugging: 
```suggestion
      ],
      "documentation": "https://headwayapp.co/bonsai-updates/254093",
```